AnnoDesigner/model/LayoutObject.cs

```diff
@@ -0,0 +1,391 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Media;
+using AnnoDesigner.Core.Models;
+
+namespace AnnoDesigner.model
+{
+    public class LayoutObject
+    {
+        private AnnoObject _wrappedAnnoObject;
+        private readonly ICoordinateHelper _coordinateHelper;
+
+        private Color? _transparentColor;
+        private SolidColorBrush _transparentBrush;
+        private Color? _renderColor;
+        private SolidColorBrush _renderBrush;
+        private Pen _borderlessPen;
+        private int _gridSizeScreenRect;
+        private Point _position;
+        private Rect _screenRect;
+        private Rect _collisionRect;
+        private Size _collisionSize;
+        private string _iconNameWithoutExtension;
+        private string _identifier;
+        private Size _size;
+        private FormattedText _formattedText;
+        private CultureInfo _usedTextCulture;
+        private Typeface _usedTextTypeFace;
+        private Rect _iconRect;
+        private int _gridSizeIconRect;
+        private Rect _lastScreenRectForIcon;
+        private Point _screenRectCenterPoint;
+        private EllipseGeometry _influenceCircle;
+        private double _influenceCircleRadius;
+        private Rect _lastScreenRectForCenterPoint;
+        private double _screenRadius;
+        private int _lastGridSizeForScreenRadius;
+        private SerializableColor _color;
+
+        public LayoutObject(AnnoObject annoObjectToWrap, ICoordinateHelper coordinateHelperToUse)
+        {
+            WrappedAnnoObject = annoObjectToWrap;
+            _coordinateHelper = coordinateHelperToUse;
+        }
+
+        public AnnoObject WrappedAnnoObject
+        {
+            get { return _wrappedAnnoObject; }
+            private set { _wrappedAnnoObject = value; }
+        }
+
+        public Color TransparentColor
+        {
+            get
+            {
+                if (_transparentColor == null)
+                {
+                    var tmp = WrappedAnnoObject.Color.MediaColor;
+                    tmp.A = 128;
+
+                    _transparentColor = tmp;
```

Comment:
I don't think so. This color is used when drawing the influence.
So if the color of an object (building) already has a transparency and we do not adjust it to "our" fixed value, it could look pretty weird.
Also it would have been a change in behaviour/apparence.